.swiftlint.yml

```diff
@@ -5,6 +5,7 @@ excluded:
   - Dependencies
   - Sources/StreamChatUI/Generated
   - Sources/StreamChatUI/StreamSwiftyGif
+  - Sources/StreamChatUI/StreamSwiftyMarkdown
```

Comment:
We need. to update the documentation to add the new dependency here: https://getstream.io/chat/docs/sdk/ios/#dependencies 🙂